src/components/custom/account/GetAddressInfo.vue

```diff
@@ -0,0 +1,223 @@
+<script setup lang="ts">
+import { Button } from '@/components/ui/button'
+import { CardContent, Card, CardHeader, CardTitle } from '@/components/ui/card'
+import { FormControl, FormField, FormItem, FormLabel, FormMessage } from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { useToast } from '@/components/ui/toast'
+import { formatEthBalance } from '@/components/utils/formatWei'
+import { useTrinConfig } from '@/composables/useTrinConfig'
+import { invoke } from '@tauri-apps/api/core'
+import { toTypedSchema } from '@vee-validate/zod'
+import { Loader2,Copy } from 'lucide-vue-next'
+import { useForm } from 'vee-validate'
+import { computed, ref } from 'vue'
+import * as z from 'zod'
+import { ref as vueRef } from 'vue'
+
+
+const { config } = useTrinConfig()
+const { toast } = useToast()
+const isLoading = ref(false)
+
+const formSchema = toTypedSchema(
+  z.object({
+    address: z
+      .string()
+      .length(42)
+      .regex(/^0x[a-fA-F0-9]{40}$/, {
+        message: "Address must be a 42-character hexadecimal string starting with '0x'"
+      }),
+    blockNumber: z
+      .string()
+      .regex(/^(0x[0-9a-fA-F]+|[0-9]+|latest|earliest|pending)$/, {
+        message: "Block number must be a number, hex value, or 'latest'/'earliest'/'pending'"
+      })
+      .default('latest')
+  })
+)
+
+const form = useForm({
+  validationSchema: formSchema
+})
+const showBytecode = vueRef(false)
+const showFullBytecode = vueRef(false)
+const accountData = ref({
+  balance: null,
+  nonce: null,
+  code: null
+})
+
+const toggleBytecode = () => {
+  showBytecode.value = !showBytecode.value
+}
+
+const toggleFullBytecode = () => {
+  showFullBytecode.value = !showFullBytecode.value
+}
+
+const copyBytecodeToClipboard = async () => {
+  try {
+    await navigator.clipboard.writeText(accountData.value.code)
+    toast({
+      title: 'Bytecode copied to clipboard',
+      variant: 'default'
+    })
+  } catch (error) {
+    toast({
+      title: 'Failed to copy bytecode',
+      description: error.toString(),
+      variant: 'destructive'
+    })
+  }
+}
+
+const onSubmit = form.handleSubmit(async (values) => {
+  isLoading.value = true
+  try {
+    const [balance, nonce, code] = await Promise.all([
+      invoke('eth_getBalance', {
```

Comment:
I have a feeling this isn't working, since the `invoke` function requires the respective functions to be set in the [invoke handler](https://github.com/njgheorghita/trin-desktop/blob/master/src-tauri/src/lib.rs#L85) ... and those functions need to be written in rust (eg. `/trin-desktop/src-tauri/src/commands/eth.rs`)

Reply:
Makes sense. I had my LLM friends code up the `eth_getCode` function in that location and it seems to be working. Would love for you to double check the work though to make sure it's logical 👀